erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -871,14 +871,9 @@ def get_exchange_rate(posting_date, account=None, account_currency=None, company
 		if reference_type in ("Sales Invoice", "Purchase Invoice") and reference_name:
 			exchange_rate = frappe.db.get_value(reference_type, reference_name, "conversion_rate")
 
-		elif account_details and account_details.account_type == "Bank" and \
-			((account_details.root_type == "Asset" and flt(credit) > 0) or
-				(account_details.root_type == "Liability" and debit)):
-			exchange_rate = get_average_exchange_rate(account)
-
 		# The date used to retreive the exchange rate here is the date passed
 		# in as an argument to this function.
-		if not exchange_rate and account_currency and posting_date:
+		elif account_currency and posting_date:
```

Comment:
Why removed `not exchange_rate`? User will not be able to set exchange_rate manually.